### DISCUSSION TITLE
Difference between Kundu's MEICA v2.5 and latest tedana

### DISCUSSION THREAD

**Discussion by Avijit-Chowdhury1** [ME-ICA/tedana#1127] [Q&A]
Hi experts,

Assuming all pre-combination preprocessing is done identically, and ```--tree kundu``` is used with other default settings, I would like to confirm if the difference between Kundu's MEICA v2.5 and the latest tedana is limited to the below steps:

(1) An adaptive mask is applied before data combination, with the threshold for each echo set as the signal of the voxel at the 33rd percentile in the shortest echo, divided by 3.
(2) PCA components are selected based on a moving average (stationary Gaussian) process (Li et al., 2007). Under default settings, no PCA component is retained based on kappa/rho (as is done in Kundu's MEICA).

Thank you for the help!

**Comment by tsalo** [comment]
It looks like this is a duplicate of https://neurostars.org/t/difference-between-prantik-kundus-meica-v2-5-and-latest-tedana/30452 so I'm going to close it.
